editoast/openapi.yaml

```diff
@@ -8686,6 +8671,25 @@ paths:
       summary: Returns the BBoxes (geo and schematic) of a line
       tags:
       - infra
+  /infra/{infra_id}/lock/:
+    post:
+      description: Lock an infra
+      parameters:
+      - description: The ID of the infra to fix
```

Comment:
Better change the comment in InfraIdParam to `An existing infra ID`

Reply:
Done.